dm/dm/master/openapi_controller.go

```diff
@@ -443,14 +444,29 @@ func (s *Server) deleteTask(ctx context.Context, taskName string, force bool) er
 	}
 	defer release()
 
+	ignoreCannotConnectError := func(err error) bool {
+		if err == nil {
+			return true
+		}
+		if force && strings.Contains(err.Error(), "connect: connection refused") {
+			log.L().Warn("connect downstream error when fore delete task", zap.Error(err))
+			return true
+		}
+		return false
+	}
+
 	toDBCfg := config.GetTargetDBCfgFromOpenAPITask(task)
 	if adjustErr := adjustTargetDB(ctx, toDBCfg); adjustErr != nil {
-		return adjustErr
+		if !ignoreCannotConnectError(adjustErr) {
+			return adjustErr
+		}
 	}
 	metaSchema := *task.MetaSchema
 	err = s.removeMetaData(ctx, taskName, metaSchema, toDBCfg)
 	if err != nil {
-		return terror.Annotate(err, "while removing metadata")
+		if !ignoreCannotConnectError(err) {
```

Comment:
I understand forcing to delete tasks shouldn't have returned errors. But what if the user restarts the same task again but the metadata still exists?

Reply:
i make a discussion note here. 
when user recreates the same task, and the downstream recovers again, it will still have the metadata, then it need remove metadata when start task or manually delete metadata. 
after disscuss with @sunzhaoyang, this is rare, we can just print warning log. but it still has risk，and most of openapi's interface has no warning message when it can run successfully. so i will pose a issue to note this.